cyclonedx_py/parser/pipenv.py

```diff
@@ -69,12 +82,15 @@ class PipEnvFileParser(PipEnvParser):
 
     def __init__(
             self, pipenv_lock_filename: str, use_purl_bom_ref: bool = False,
+            omit_category: Set[str] = set(),  # noqa: B006
```

Comment:
:x: do NOT ignore/suppress the bug.
FIX IT! 

this is a serious issue. Please learn how to avoid the "bug" 
and fix the implementation.